### PR TITLE
[VTCCA-3006] - fixed back link on declaration page

### PR DESCRIPTION
**Pull request by jon-pike**

### CODE REVIEW

File: app/controllers/propertyLinking/Declaration.scala [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Declaration @Inject()(
   val logger = Logger(this.getClass.getName)
 
   def show(): Action[AnyContent] = authenticatedAction.andThen(withLinkingSession) { implicit request =>
-    val isRatesBillEvidence = request.ses.evidenceType.contains(RatesBillType)
+    val isRatesBillEvidence = request.ses.uploadEvidenceData.linkBasis.name == "RATES_BILL"
```
Comment (darrell-plant) [inline app/controllers/propertyLinking/Declaration.scala]: Isn't `request.ses.uploadEvidenceData.linkBasis == RatesBillType` better?

Comment (jon-pike) [reply app/controllers/propertyLinking/Declaration.scala]: yep - Thanks Darrell

**Comment by darrell-plant** [comment]
👍

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**